extension/src/service/web-component-service.js

```diff
@@ -94,7 +94,7 @@ function getCarbonOffsetCosts(merchantAccount, getCarbonOffsetCostsRequestObj) {
     `${adyenCredentials.apiBaseUrl}/carbonOffsetCosts`,
     merchantAccount,
     adyenCredentials.apiKey,
-    extendRequestObjWithApplicationInfo(getCarbonOffsetCostsRequestObj)
+    getCarbonOffsetCostsRequestObj
```

Comment:
Why did you make this change?

Reply:
carbonOffsetCosts endpoint does not accept applicationInfo field 🤷🏽

Reply:
Can you inform Adyen about this? I guess they want to have this info sent to them.